back/engines/commercial/multi_tenancy/lib/tasks/core/tenant_settings.rake

```diff
@@ -23,9 +24,76 @@ namespace :tenant_settings do
     failed = []
     Tenant.where.not(host: 'i18n.stg.citizenlab.co').each do |tn|
       Apartment::Tenant.switch(tn.schema_name) do
-        tn.settings[args[:feature]]['allowed'] = false
-        tn.settings[args[:feature]]['enabled'] = false
-        failed += [tn.host] unless tn.save
+        config = AppConfiguration.instance
+        config.settings[args[:feature]]['allowed'] = false
+        config.settings[args[:feature]]['enabled'] = false
+        failed += [tn.host] unless config.save
+      end
+    end
+    if failed.present?
+      puts "Failed for: #{failed.join(', ')}"
+    else
+      puts 'Success!'
+    end
+  end
+
+  # Usage:
+  #   bundle exec rake tenant_settings:enable_feature_for_tenants[feature,url]
+  # Example:
+  #   bundle exec rake tenant_settings:enable_feature_for_tenants['participatory_budgeting','/tenants.txt']
```

Comment:
Will this `tenants.txt` file be deleted after deployment? How will we restore it from the UI?